### ISSUE TITLE
inconsistent tensor size, expected tensor [31 x 31] and src [30 x 30] to have the same number of elements, but got 961 and 900 elements respectively at /tmp/luarocks_torch-scm-1-4200/torch7/lib/TH/generic/THTensorCopy.c:86

### ISSUE THREAD

**Issue by saisrinivas047** [facebookresearch/wav2letter#38]
getting this error when I am using the pretrained model to transcribe my own .flac files. Can someone help me solve this

"inconsistent tensor size, expected tensor [31 x 31] and src [30 x 30] to have the same number of elements, but got 961 and 900 elements respectively at /tmp/luarocks_torch-scm-1-4200/torch7/lib/TH/generic/THTensorCopy.c:86"

**Comment by vineelpratap** [comment]
Can you let us know the procedure followed, the exact command you have run and the complete error log. 

Looking at the error, it looks like you have not created letters.lst file correctly. Have you completed this step - https://github.com/facebookresearch/wav2letter#running-the-decoder-inference

**Comment by saisrinivas047** [comment]
hi thanks for the reply
first i created a folder inside LibriSpeech directory and added one flac audio file there. then I removed all the subfolderpaths from create.lua and added my new folder path there. then I followed the data pre-processing steps and decoding steps. and finally I ran this command:
"luajit test.lua librispeech-glu-highdropout-cpu.bin -progress -show -test dev-test -save -datadir librispeech-proc/ -dictdir librispeech-proc/ -gfsai"

is it necessary to run theses commands also:
"luajit ~/wav2letter/test.lua ~/experiments/hello_librispeech/001_model_dev-clean.bin -progress -show -test dev-clean -save" and 
luajit ~/wav2letter/decode.lua ~/experiments/hello_librispeech dev-clean -show -letters ~/librispeech-proc/letters-rep.lst  -words ~/dict.lst -lm ~/3-gram.pruned.3e-7.arpa -lmweight 3.1639 -beamsize 25000 -beamscore 40 -nthread 10 -smearing max -show
